src/main/java/org/sagebionetworks/web/client/widget/table/v2/TableEntityWidget.java

```diff
@@ -66,6 +74,23 @@ public static final String getNoColumnsNotEditableMessage(TableType tableType) {
 		return "This " + tableType.getDisplayName() + " does not have any columns.";
 	}
 
+	public static final String noScopeEditableMessage(TableType tableType) {

```

Comment:
Append to noScopeNotEditableMessage()? Benefit is that it would remove duplicate copy.